src/distances.jl

```diff
@@ -24,53 +32,76 @@ julia> Ellipsoidal([1.0,0.5], [π/2])
 julia> Ellipsoidal([1.0,0.5,0.5], [π/2,0.0,0.0])
 ```
 """
+
+# Right-hand rule; motion defined looking to the negative direction of the axis
+struct RotationRules
+  rot_seq::Symbol
+  motion::Vector{Symbol}
+  radian::Bool
+  main::Symbol
+  extrinsic::Bool
+end
+
+themes = Dict(
+  :TaitBryanExtr => RotationRules(:ZXY,[:CCW,:CCW,:CCW],true,:x,true),
+  :TaitBryanIntr => RotationRules(:ZXY,[:CCW,:CCW,:CCW],true,:x,false),
+  :EulerExtr => RotationRules(:ZXZ,[:CCW,:CCW,:CCW],true,:x,true),
+  :EulerIntr => RotationRules(:ZXZ,[:CCW,:CCW,:CCW],true,:x,false),
+  :GSLIB => RotationRules(:ZXY,[:CW,:CCW,:CCW],false,:y,false),
+  :Leapfrog => RotationRules(:ZYZ,[:CW,:CW,:CW],false,:x,false),
+  :Datamine => RotationRules(:ZXZ,[:CW,:CW,:CW],false,:x,false)
```

Comment:
It would be nice to align the arrows of the dict for readability.

Reply:
Done